Content.Server/GameObjects/Components/Arcade/BlockGameArcadeComponent.cs

```diff
@@ -538,36 +538,46 @@ private void InitializeNewBlock(BlockGamePiece piece)
 
             public void ProcessInput(BlockGamePlayerAction action)
             {
+                if (_running)
+                {
+                    switch (action)
+                    {
+                        case BlockGamePlayerAction.StartLeft:
+                            _leftPressed = true;
+                            break;
+                        case BlockGamePlayerAction.EndLeft:
+                            _leftPressed = false;
+                            break;
+                        case BlockGamePlayerAction.StartRight:
+                            _rightPressed = true;
+                            break;
+                        case BlockGamePlayerAction.EndRight:
+                            _rightPressed = false;
+                            break;
```

Comment:
same as with SoftDropEnd

Reply:
Suggestions applied, thanks for the review.